EiDiA_backend/src/controllers/record.js

```diff
@@ -54,8 +55,54 @@ const addRecord = (req, res) => {
             }
         });
 };
+const listDocumentsByTypeAndRecord = (req, res) => {
+
+    DocumentModel.find(req.body.mapping.map(mapping => {
+            return {documentTypeId: mapping.docTypeId, recordId: req.body.recordId}
+        }
+    ))
+        .then(documentList => {
+            res.status(200).json({documentList: documentList});
+        })
+        .catch(error => {
+
+            res.status(500).json({
+                error: 'Internal server error',
+                message: error.message,
+            });
+
+        });
+}
+
+const listDocumentByRecordId = (req, res) => { // Return attributes based on selected DocumentTypeId
+    DocumentModel.find({'recordId': mongoose.Types.ObjectId(req.params.recordId)})
+        .then(documentList => {
+            let response = documentList.map(document => {
+                return {
+                    id: document._id,
+                    name: document.name,
+                    parentFolderId: document.parentFolderId,
+                    fileType: document.fileType, //TODO: implement filetype in database
```

Comment:
```suggestion
                    fileType: document.fileType,
```
an der Stelle ist auch in Zukunft nichts zu tun